source/docs/ondemand_rds_vhds.md

```diff
@@ -0,0 +1,161 @@
+# On-Demand RDS/VHDS
+Currently, in RDS, all routes for the cluster are sent to every Envoy instance in the mesh. This causes scaling issues as the size of the cluster grows. The majority of this complexity can be found in the virtual host configurations, of which most are not needed by any individual sidecar. With a goal of being able to scale to one million vhosts in the future, the capability to filter a sidecar to only contain the required virtual hosts is a necessity. 
```

Comment:
The use case for VHDS goes beyond sidecars, so maybe replace "sidecar" with "proxy".

Reply:
Done. Thanks